src/main/java/com/epam/preprod/sirenko/command/CommandContainer.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.preprod.sirenko.command;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * All commands holder
+ */
+public class CommandContainer {
+	private static Map<String, Command> commands = new TreeMap<>();
```

Comment:
Please try to avoid using static methods if possible

Reply:
done